ADR/0011-roles-and-permissions.md

```diff
@@ -25,6 +25,7 @@ We will use the built-in Kubernetes RBAC system for Stonesoup's role and permiss
 | Contributor   | Workspace               | Access to namespaces that backs workspace                           |
 |               | Application & Component | appstudio.redhat.com      | get, list, watch                        | applications, components, componentdetectionqueries
 |               | Environment             | appstudio.redhat.com      | get, list, watch                        | promotionruns, snapshotenvironmentbindings, snapshots, environments
+|               | DeploymentTarget       | appstudio.redhat.com      | get, list, watch                        | deploymentttargets, deploymenttargetclaims, deploymenttargetclasses
```

Comment:
I use `DeploymentTarget` as the value of `Permissions`, should we list the all three CRDs in the `Permissions`?